public/app/core/components/AppChrome/News/NewsContainer.tsx

```diff
@@ -1,18 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useToggle } from 'react-use';
 
-import { Drawer, ToolbarButton } from '@grafana/ui';
+import { Drawer, ToolbarButton, useTheme2 } from '@grafana/ui';
+import { useMediaQueryChange } from 'app/core/hooks/useMediaQueryChange';
 import { DEFAULT_FEED_URL } from 'app/plugins/panel/news/constants';
 
 import { NewsWrapper } from './NewsWrapper';
 
 export function NewsContainer() {
+  const theme = useTheme2();
   const [showNewsDrawer, onToggleShowNewsDrawer] = useToggle(false);
+  const breakpoint = theme.breakpoints.values.sm;
+
+  const [isSmallScreen, setIsSmallScreen] = useState(window.matchMedia(`(max-width: ${breakpoint}px)`).matches);
```

Comment:
don't think there's any need for a js media query here. also don't think it should be a general property of `NewsContainer` - it's logic that's specific to the `TopSearchBar` (i.e. we want the `TopSearchBar` to hide the `News` button on small screens). imagine if we want to use this button elsewhere... now it will disappear on a small screen anywhere we use it 😱 

think you could expose a new optional `className` prop, pass it to `ToolbarButton`, then define a `newsButton` class in `TopSearchBar` that looks like this:
```typescript
newsButton: css({
  [theme.breakpoints.down('sm')]: {
    display: 'none',
  }
}),
```